Cargo.toml

```diff
@@ -24,12 +24,16 @@ opt-level = 's'
 #default = ["wee_alloc"]
 
 [dependencies]
-rhai = { version = "1.7.0", features = [ "only_i64", "no_optimize", "internals" ] }
+rhai = { version = "1.12.0", features = [
+  "only_i64",
+  "no_optimize",
+  "internals",
+] }
 # rhai = { version = "0.19.16", features = [ "only_i64", "no_optimize", "internals" ], git = "https://github.com/rhaiscript/rhai.git", branch = "master" }
 js-sys = "0.3"
 serde = { version = "1.0", features = ["derive"] }
 instant = { version = "0.1", features = ["wasm-bindgen"] }
-
+getrandom = { version = "0.2", features = ["js"] }
```

Comment:
I believe it is now possible to put `wasm-bindgen` feature on `rhai`, such that this `getrandom` is no longer necessary.

Reply:
@schungx done in [882d9d7](https://github.com/rhaiscript/playground/pull/23/commits/882d9d7b96d7db91bc06077b0c135ff807ce3454)

You can see it in action here: https://fabriziosestito.github.io/playground/dev/

Should I rebase and fix conflicts, it might fix the CI as well.